packages/office-ui-fabric-react/src/components/Calendar/docs/CalendarOverview.md

```diff
@@ -1 +1,6 @@
 The calendar component allows a user to browse through a calendar and pick a date value.
+
+This component is being deprecated in favor of @uifabric/date-time's `Calendar` component.
+https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/date-time/src/components/Calendar/docs/CalendarOverview.md
+
+The newer component has several bug fixes in it, as well as support for the `styles` parameter for customization using merge-styles.
```

Comment:
I added the note about visual design because the site is also used by designers (not just developers). If there are actually some visual differences, feel free to update accordingly.